pysrc/bytewax/testing.py

```diff
@@ -158,3 +169,59 @@ def poll_next_batch(source: StatefulSource, timeout=timedelta(seconds=5)):
             raise TimeoutError()
         batch = source.next_batch()
     return batch
+
+
+def _parse_args():
+    parser = _create_arg_parser()
+
+    # Add scaling arguments for the testing namespace
+    scaling = parser.add_argument_group(
+        "Scaling",
+        "This testing entrypoint supports using '-p' to spawn multiple ",
+        "processes, and '-w' to run multiple workers within a process.",
```

Comment:
remove the first comma here:
```suggestion
        "This testing entrypoint supports using '-p' to spawn multiple "
        "processes, and '-w' to run multiple workers within a process.",
```